cdc/owner/feed_state_manager.go

```diff
@@ -423,6 +423,21 @@ func (m *feedStateManager) handleError(errs ...*model.RunningError) {
 		}
 	}
 
+	for _, err := range errs {
```

Comment:
Please add a comment here explaining why we are doing this or why we need to do this.